Added: alt attribute text to few images (#19696)
mdn/content: files/en-us/web/api/canvas_api/tutorial/compositing/index.md

A clipping path is like a normal canvas shape but it acts as a mask to hide unwanted parts of shapes. This is visualized in the image below. The red star shape is our clipping path. Everything that falls outside of this path won't get drawn on the canvas.

> ![A canvas co-ordinate with a star outlined in red color. The grid squares lying inside the star are clearly visible whereas the grid squares lying outside the star are blurred. The center of the canvas is at the left-hand side top corner. X-axis is the horizontal line at the top. Y-axis is the horizontal line at the left.](canvas_clipping_path.png)

Reviewer: ```suggestion
![A canvas with a star outlined in red color. The inside of the star is transparent, as portrayed by the grid squares inside the star being clearly visible whereas the grid squares lying outside the star are blurred. ](canvas_clipping_path.png)
```
